docs/DeveloperGuide.md

```diff
@@ -12,33 +12,49 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
-### Study Manager component
-API: `StudyManager.java`
-
-![](images/StudyManagerClass.png)
 
-The `StudyManager` component,
-1. Stores the academic schedule, i.e. all `Module` objects are contained in a `modulesList` object.
-2. The storage can be bound to `modulesList` list such that everytime a change is observed in the list it is saved on to a file.
-3. Does not depend on other components like `ExpenseTracker` and `ContactsManager`
 
 
 
 ## Product scope
 ### Target user profile
```

Comment:
Thanks for including this! Perhaps include another line or two about our app targeting NUS students?

Reply:
Sure